wob.c

```diff
@@ -654,9 +663,8 @@ main(int argc, char **argv)
 					return EXIT_SUCCESS;
 				}
 
-				old_background_color = background_color;
-				old_border_color = border_color;
-				if (fgets_rv == NULL || !wob_parse_input(input_buffer, &percentage, &background_color, &border_color, &bar_color) || percentage > maximum) {
+                old_colors = colors;
```

Comment:
Tabs please :slightly_smiling_face:

Reply:
Ah, sorry about that dude totally forgot to run clang-format  :man_facepalming: . Should all be fixed now